tasks/administration/install/script.nix

```
@@ -0,0 +1,14 @@
+{ inputs, ... }:
+
+# Task to INSTALL the specified derivation on current system including the firmware in a fully declarative way
+
+# Refer to https://github.com/nix-community/disko/issues/657#issuecomment-2146978563 for implementation notes
+
+{
+	perSystem = { system, pkgs, inputs', ... }: {
+		install-task = pkgs.writeShellScriptBin "install-task" ''
```

```suggestion
		packages.install-task = pkgs.writeShellScriptBin "install-task" ''
```

(See https://flake.parts/options/flake-parts#opt-perSystem.packages)

If this is more something to run than something to install, I suppose you could define it as a flake app (`nix run` only)

```suggestion
		apps.install-task.command = pkgs.writeShellScriptBin "install-task" ''
```

See https://flake.parts/options/flake-parts#opt-perSystem.apps._name_.program

---

@roberth Thanks a lot for help! <3

Using that as an app to be able to use `nix run` to just deploy the system without having to clone the repository is very functional, i will brainstorm with other NiXium devs to figure out how to implement it in the repository.